### DISCUSSION TITLE
How do I read resource logs in a terminal?

### DISCUSSION THREAD

**Discussion by KennethHoff** [microsoft/aspire#6186] [Q&A] [answered]
I understand that a major selling point of Aspire is the Dashboard - and it is good - but I was wondering if it is possible to view the console logs for a specific service in a terminal?

Ideally there would be some kind of `dotnet aspire --attach --resource=<resourceName> log` or something that allowed you to read console logs for specific resources inside a CLI

**Comment by davidfowl** [comment, marked as answer]
> I understand that a major selling point of Aspire is the Dashboard - and it is good - but I was wondering if it is possible to view the console logs for a specific service in a terminal?

No this doesn't exist. 

PS: We do have a feature to expose resource logs in the aspire.hosting output but it's not exposed in a clean way. That's what https://github.com/dotnet/aspire/pull/5878 is about.

**Reply by KennethHoff** [reply]
Does this mean that, with the plausible introduction of `ResourceLoggerForwarderService`, one could create their own Aspire Resource that reads console logs of a resource, and outputs it to some endpoint that we then could read via the terminal?

**Reply by davidfowl** [reply]
Yes
